lang/talon/talon.talon

```diff
@@ -4,9 +4,10 @@ mode: command
 and mode: user.auto_lang
 and code.language: talon
 -
-tag(): user.code_operators
-tag(): user.code_comment
-# uncomment user.talon_populate_lists tag to activate talon-specific lists of actions, scopes, modes etcetera. 
+tag(): user.code_operators_math
```

Comment:
With this change .talon files lose the 'op (equals | assign)' command. Would just need to add in the tag(): user.code_operators_assignment tag here.